src/convertToObject.js

```diff
@@ -6,7 +6,27 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylePropertiesSplittedByDotComa = sourceString
+    .split(';')
+    .map((item) => item.trim())
+    .filter((item) => item.length > 0);
+
+  const stylePropertiesSplittedByDoubleDot = stylePropertiesSplittedByDotComa
+    .map((item) => item.split(':'))
+    .map((SplittedCssProperty) =>
+      // eslint-disable-next-line prettier/prettier
+      SplittedCssProperty.map((propertyPart) => propertyPart.trim()));
```

Comment:
- You can do inside reduce method
- `SplittedCssProperty` - variable names should be in camelCase instead of PascalCase

Reply:
yea, I changed variable name (small typo), but I didn't put in _reduce_ 'cause it looks like a porridge and adds more complexity